From 5e6af8c15632e2e9acbf86d18a3807230d4636d1 Mon Sep 17 00:00:00 2001
From: Huiquan Zhong <huiquan.zhong@intel.com>
Date: Fri, 16 Jan 2015 16:02:38 +0800
Subject: [PATCH] dma: dw: delelte unused variable 'i'

Delete unused variable 'i' in dw_dma_off().

Change-Id: I7bad310719c1e49416d4d07ea3f8b8ffca7fe903
Signed-off-by: Huiquan Zhong <huiquan.zhong@intel.com>
(cherry picked from commit f5bcda5327f6c7068939bd60bbd09c4f63612b2f)

Signed-off-by: Brett T. Warden <brett.t.warden@intel.com>
---
 drivers/dma/dw/core.c |    2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/dma/dw/core.c b/drivers/dma/dw/core.c
index 1e23352..4bbbc1c 100644
--- a/drivers/dma/dw/core.c
+++ b/drivers/dma/dw/core.c
@@ -1475,8 +1475,6 @@ EXPORT_SYMBOL(dw_dma_cyclic_free);
 
 static void dw_dma_off(struct dw_dma *dw)
 {
-	int i;
-
 	dma_writel(dw, CFG, 0);
 
 	channel_clear_bit(dw, MASK.XFER, dw->all_chan_mask);
-- 
1.7.9.5

